tests/lib/rules/array-callback-return.js

```diff
@@ -14,11 +14,22 @@ const rule = require("../../../lib/rules/array-callback-return"),
 
 const ruleTester = new RuleTester();
 
+const options = [{ allowImplicit: true }];
```

Comment:
This can be made clearer by naming it `allowImplicitOptions` or something similar. I know that prevents the use of the shorthand object syntax below, but it makes it unclear what this is if it's just called `options`. It almost seems like it's the default!